### PR TITLE
Implement MetadataLoggerPlug

### PR DESCRIPTION
**Pull request by bettio**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -21,8 +21,7 @@ defmodule MetadataLoggerPlug.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      # {:dep_from_hexpm, "~> 0.3.0"},
-      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+      {:plug, "~> 1.4.0"}
```
Comment (rbino) [inline mix.exs]:
```
{:plug, "~> 1.4"}
```
otherwise it will complain if you use any minor other than `1.4`

File: lib/metadata_logger_plug.ex [hunk 2]
```diff
@@ -1,18 +1,56 @@
+# Based on the original Plug.Logger [1] but focused on metadata Logging.
```
Comment (rbino) [inline lib/metadata_logger_plug.ex]: Move this in moduledoc

File: lib/metadata_logger_plug.ex [hunk 3]
```diff
@@ -1,18 +1,56 @@
...
+
 defmodule MetadataLoggerPlug do
   @moduledoc """
-  Documentation for MetadataLoggerPlug.
-  """
+  A plug for logging basic request information in the format:
 
-  @doc """
-  Hello world.
+      Got client request.
```
Comment (rbino) [inline lib/metadata_logger_plug.ex]: Is this the actual format?

Comment (bettio) [reply lib/metadata_logger_plug.ex]: yes

**Pull request merged**